aspnetcore/mvc/advanced/app-parts/3.0sample1/WebAppParts/StartupRm.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System.Linq;
+
+
+namespace WebAppParts
+{
+    public class StartupRm
+    {
+        public StartupRm(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        #region snippet
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllersWithViews()
+                    .ConfigureApplicationPartManager(apm =>
+                    {
+                        var dependentLibrary = apm.ApplicationParts
+                            .FirstOrDefault(part => part.Name == "MySharedApp");
+
+                        if (dependentLibrary != null)
+                        {
+                            apm.ApplicationParts.Remove(dependentLibrary);
```

Comment:
Can we remove this sample? I'm not sure what value there is in showing users ways to remove things from application discovery.